cmd/simulator/load/loader.go

```diff
@@ -96,24 +106,28 @@ func CreateLoader(ctx context.Context, config config.Config) (*WorkerGroup, erro
 	}
 	txSequences, err := GenerateTxSequences(ctx, txGenerator, clients[0], pks, config.TxsPerWorker)
```

Comment:
Do you think `Agent` can handle generating tx sequences itself by taking priv keys + numTx?